src/04_printing.py

```diff
@@ -11,7 +11,8 @@
 # Using the printf operator (%), print the following feeding in the values of x,
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
-
+print('x is %d, y is %.2f, z is "%s"' %(x, y, z))
 # Use the 'format' string method to print the same thing
-
-# Finally, print the same thing using an f-string
\ No newline at end of file
+print('x is {}, y is {:0.2f}, z is "{}" '.format(x, y, z))
+# Finally, print the same thing using an f-string
+print(f'x is {x}, y ix {y:.2f}, z is "{z}"')
```

Comment:
Good implementations! You can also use the escaping char `\` like so `print("x is {}, y is {:.2f}, z is \"{}\"".format(x, y, z))`

Reply:
Thank you.